eq3bt/eq3cli.py

```diff
@@ -22,16 +22,17 @@ def validate_mac(ctx, param, mac):
 @click.group(invoke_without_command=True)
 @click.option('--mac', envvar="EQ3_MAC", required=True, callback=validate_mac)
 @click.option('--interface', default=None)
+@click.option('--connectionattempts', default=2)
```

Comment:
```suggestion
@click.option('--retry', default=2)
```
Or `--retries` would be shorter and still understandable name for this.

Reply:
That's okay for me. But 'connectionattempts' is not the same as 'retries'. The change must also be reflected in eq3btsmart.py and connection.py. See commit 603b391